cvat/apps/consensus/consensus_reports.py

```diff
@@ -0,0 +1,493 @@
+# Copyright (C) 2024 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from __future__ import annotations
+
+import itertools
+from collections import Counter
+from copy import deepcopy
+from functools import cached_property
+from types import NoneType
+from typing import Any, Dict, List, Optional, cast
+
+import datumaro as dm
+import numpy as np
+from attrs import define, fields_dict
+from datumaro.components.annotation import Annotation
+from datumaro.util import dump_json, parse_json
+from django.db import transaction
+from django.utils import timezone
+
+from cvat.apps.consensus import models
+from cvat.apps.consensus.models import (
+    AssigneeConsensusReport,
+    ConsensusConflict,
+    ConsensusConflictType,
+    ConsensusReport,
+    ConsensusSettings,
+)
+from cvat.apps.consensus.new_intersect_merge import IntersectMerge
+from cvat.apps.dataset_manager.util import bulk_create
+from cvat.apps.engine.models import Job, Task, User
+from cvat.apps.quality_control.quality_reports import AnnotationId, JobDataProvider, _Serializable
+
+
+@define(kw_only=True)
+class AnnotationConflict(_Serializable):
```

Comment:
```suggestion
class AnnotationConflict(Serializable):
```
Please rename so that it wasn't a private symbol access.